docs/content/en/docs/getting-started/developer/_index.md

```diff
@@ -0,0 +1,215 @@
+        "message": "DataStore is operational",
+        "duration": 3,
+        "timestamp": "2021-02-24T19:06:57.450Z"
+      },
+      "deadlocks": {
+        "healthy": true,
+        "duration": 1,
+        "timestamp": "2021-02-24T19:06:57.452Z"
+      },
+      "graphql": {
+        "healthy": true,
+        "message": "Ready to process requests",
+        "duration": 0,
+        "timestamp": "2021-02-24T19:06:57.450Z"
+      },
+      "restapi": {
+        "healthy": true,
+        "message": "Available",
+        "duration": 0,
+        "timestamp": "2021-02-24T19:06:57.450Z"
+      },
+      "schema-agreement": {
+        "healthy": true,
+        "message": "All schemas agree",
+        "duration": 0,
+        "timestamp": "2021-02-24T19:06:57.450Z"
+      }
+    }
+    ```
+
+2. Generate a Stargate access token replacing `<k8ssandra-username>` and `<k8ssandra-password>` with the values you retrieved in [Retrieve K8ssandra superuser credentials]({{< relref "/docs/getting-started#superuser" >}}):
+
+    ```bash
+    curl -L -X POST 'http://localhost:8081/v1/auth' -H 'Content-Type: application/json' --data-raw '{"username": "<k8ssandra-username>", "password": "<k8ssandra-password>"}'
+    {"authToken":"<access-token>"}
+    ```
+
+3. Use `<access-token>` to populate the `x-cassandra-token` header for all Stargate requests.
+
+Once you've got the access token, take a look at the following Stargate access options:
+
+* [Access Document Data API]({{< relref "docs/topics/stargate#access-document-data-api" >}})
+* [Access REST Data API]({{< relref "docs/topics/stargate#access-rest-data-api" >}})
+* [Access GraphQL Data API]({{< relref "docs/topics/stargate#access-graphql-data-api" >}})
+
+For complete details on Stargate, see the [Stargate documentation](https://stargate.io/docs/stargate/1.0/quickstart/quickstart.html).
+
+## Access Cassandra using CQLSH
+
+If you're familiar with Cassandra, then you're familiar with CQLSH. You can download a full-featured CQLSH utility from Datastax and use that to interact with K8ssandra as if you were in a native Cassandra environment.
+
+To access K8ssandra using the stand alone CQLSH utility:
+
+1. Make sure you have [Python 2.7](https://www.python.org/download/releases/2.7/) installed on your system.
+
+1. Download CQLSH from the  [DataStax download site](https://downloads.datastax.com/#cqlsh) choosing the version for **DataStax Astra**.
+
+1. Connect to Cassandra replacing `<k8ssandra-username>` and `<k8ssandra-password>` with the values you retrieved in [Retrieve K8ssandra superuser credentials]({{< relref "/docs/getting-started#superuser" >}}):
+
+    ```bash
+    cqlsh -u <k8ssandra-username> -p <k8ssandra-password>
+    Connected to k8ssandra at 127.0.0.1:9042.
+    [cqlsh 6.8.0 | Cassandra 3.11.6 | CQL spec 3.4.4 | Native protocol v4]
+    Use HELP for help.
+    k8ssandra-superuser@cqlsh>
+   ```
+
+1. Populate a CQL file with the following data and save as `test-data.cql`:
```

Comment:
This assumes the user exits `cqlsh`(and that they know how to exit), and that they relaunch `cqlsh`. Maybe they should just execute the commands since they're already at the prompt. Sourcing a file with just a few commands seems unnecessary.

Reply:
Yeah, I think I'll go through the create keyspace, create table, add data individually and then do a quit;